libp2p/switch.nim

```diff
@@ -510,8 +510,12 @@ proc stop*(s: Switch) {.async.} =
     except CatchableError as exc:
       warn "error cleaning up transports", msg = exc.msg
 
-  let stopped = await allFuturesThrowing(s.acceptFuts)
-    .withTimeout(1.seconds)
+  var stopped: bool
```

Comment:
```
var stopped = try: 
    await all... 
  except ...: 
    trace "..."
    false
```

Reply:
this style makes the compiler ensure that all branches get an explicit value for `stopped`